packages/vuetify/src/components/VDatePicker/util/__tests__/determineWeeknumber.spec.ts

```diff
@@ -0,0 +1,30 @@
+import determineWeeknumber from '../determineWeeknumber'
+
+describe('VDatePicker/util/determineWeeknumber.ts', () => {
+  it('should return 53 as weeknumber', () => {
```

Comment:
i don't think it's necessary to split examples into separate tests, maybe just merge them together and call it `it('should return valid week number', ...)`
Also if the `firstDayOfWeek` argument matters there could be test showing different week numbers for different values of this argument

Reply:
Combined all tests to one case